wrangler-core/src/main/java/io/cdap/wrangler/expression/EL.java

```diff
@@ -117,6 +115,24 @@ public ELResult execute(ELContext context, boolean nullMissingFields) throws ELE
     }
   }
 
+  private String constructErrMessage(JexlException e) {
```

Comment:
should this be used in this PR as well: https://github.com/data-integrations/wrangler/pull/407

Reply:
Also where is this function used?

Reply:
That's ErrorRecord which is not same as finding this error.